services/transcode/video.go

```diff
@@ -65,16 +60,16 @@ func VideoH264(input common.VideoInput, cb ffmpeg.ProgressCallback) (*common.Vid
 		filterComplex += "[0:0]copy[main];"
 	}
 
-	height := input.Height
-	width := -1
-	videoIsPortrait := info.Height > info.Width
-	paramsIsPortrait := input.Height > input.Width
-	if info.Height != 0 && info.Width != 0 && videoIsPortrait && !paramsIsPortrait {
```

Comment:
This logic was here because we want to flip the resolution if the video is portrait automatically.
It simplifies the triggering since we do triggering with the VOD resolutions e.g. 1920x1080

Reply:
Aaaah, I see. 
Hm, I will adjust the resolution thingy then to behave in a similar way.